OfficeToPDF/ExcelConverter.cs

```diff
@@ -410,6 +408,14 @@ public static new int Convert(String inputFile, String outputFile, Hashtable opt
 
                 workbook.SaveAs(tmpFile, fmt, Type.Missing, Type.Missing, Type.Missing, false, XlSaveAsAccessMode.xlNoChange, Type.Missing, false, Type.Missing, Type.Missing, Type.Missing);
 
+                // Fit to to one page wide.
+                if (fitToPageWidth)
+                {
+                    templatePageSetup["FitToPagesWide"] = 1;
+                    templatePageSetup["FitToPagesTall"] = false;

```

Comment:
Q: don't we also need landscape instead of portrait mode?

Reply:
Right now we don't force landscape or portrait, it just assumes the user's setting. ( landscape is the default ).

I could add the option, but don't sure if we want to make that choice for the user.

Reply:
I added it in the consequences doc so support can pick it up in their documentation